docs/expo.md

```diff
@@ -0,0 +1,45 @@
+## Guide for apps created using the Expo CLI
```

Comment:
Can we include a 'Table of Contents' section as the top level heading? Customers should be able to easily locate the topic of interest.
```
# Table of Contents
- [Heading1](#heading1)
- [Heading2](#heading2)
```